src/reducers/InputOutput.js

```diff
@@ -1,9 +1,32 @@
 import NPMsAlert from 'react-s-alert';
-
+import math from 'mathjs';
 import {exportToDisk} from "../utils/downloader";
+import {Made} from "@exabyte-io/made.js";
 import {MATERIALS_EXPORT, MATERIALS_ADD, MATERIALS_REMOVE, MATERIALS_SAVE} from "../actions";
+import {Material} from "../material";
 
 export function materialsSave(state, action) {
```

Comment:
We need to change the basis upon saving but there was not a clear `prop` to pass to the `Basis` class for materials-designer to deal with the changes. If there is a way i did not think of clearly to create a Materials-Designer Basis class object I would love to know what it is. I kept getting errors related to calling `xyz: this.props.material.getBasisAsXyz(),`